easy_phi/tests/run_tests.py

```diff
@@ -6,6 +6,7 @@
 TEST_MODULES = [
     'easy_phi.tests.handlers_test',
     'easy_phi.tests.mod_conf_patch_test',
+    'easy_phi.tests.regression_test',
```

Comment:
probably this part should not be executed as a part of unit test, since it checks performance and should be executed only for releases. Makefile is a better place to put timing test, but regression tests to check for degradation should be here